kernel/byterun/coq_values.h

```diff
@@ -28,6 +28,20 @@
 
 /* Les blocs accumulate */
 #define Is_accu(v) (Is_block(v) && (Tag_val(v) == Accu_tag))
-
 #define IS_EVALUATED_COFIX(v) (Is_accu(v) && Is_block(Field(v,1)) && (Tag_val(Field(v,1)) == ATOM_COFIXEVALUATED_TAG))
+#define Is_uint63(v) (Is_long(v) || (Tag_val(v) == Custom_tag))
```

Comment:
For efficiency, this should probably be defined differently depending on the architecture.

Reply:
Done, thanks for the suggestion.